packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactHorizontalScrollView.java

```diff
@@ -990,6 +991,34 @@ public void run() {
         this, mPostTouchRunnable, ReactScrollViewHelper.MOMENTUM_DELAY);
   }
 
+  public void handleSmoothScrollMomentumEvents() {
+    if (!mSendMomentumEvents || null != mPostSmoothScrollRunnable) {
+      return;
+    }
+
+    enableFpsListener();
```

Comment:
Commenting back on this one, we have someone working on a fix internally to ensure performance listeners are properly registered. This will be omitted from this PR but will be added later